tests/DecodeChunkedStreamTest.php

```diff
@@ -76,7 +76,10 @@ public function provideChunkedEncoding()
             ],
             'uppercase-chunk' => [
                 ["4\r\nWiki\r\n5\r\npedia\r\nE\r\n in\r\n\r\nchunks.\r\n0\r\n\r\n"],
-            ]
+            ],
+			'extra-space-in-length-chunk' => [
+				["04 \r\nWiki\r\n5\r\npedia\r\nE\r\n in\r\n\r\nchunks.\r\n0\r\n\r\n"],
+			]
```

Comment:
Minor CS issue.

Also, can you add a test a chunk header that consists of a single space `" \r\n"` will be rejected?